.github/workflows/continuous-integration.yml

```diff
@@ -82,6 +82,21 @@ jobs:
         with:
           args: "xmllint --noout --schema vendor/squizlabs/php_codesniffer/phpcs.xsd lib/Doctrine/ruleset.xml"
 
+  lint-shell-scripts:
+    name: "Lint shell scripts"
+
+    runs-on: "ubuntu-22.04"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v3"
+
+      - name: "Lint shell scripts with ShellCheck"
+        uses: "ludeeus/action-shellcheck@2.0.0"
```

Comment:
Do we need to specify the version number up to the patch version?

Reply:
I think it is required as there are no aliases ([branch](https://github.com/ludeeus/action-shellcheck/branches) or [tag](https://github.com/ludeeus/action-shellcheck/tags)) for `2.0` in that repo.